aten/src/ATen/native/native_functions.yaml

```diff
@@ -2333,11 +2341,25 @@
 - func: sinh.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   supports_named_tensor: True
 
+# Returns a copy of this `Variable` that is detached from its autograd graph
+# and has a blank version. This method is OK to call if the `Variable` is a
```

Comment:
this isn't correct, I think you are confusing `detach` and `.data` (in python).

E.g.:
```
>>> torch.randn(2,3).add_(1).add_(2).detach()._version
2
>>> torch.randn(2,3).add_(1).add_(2).data._version
0
```

Reply:
BTW, I plead "pre-existing" code ;) (I'll fix)